README.md

```diff
@@ -24,7 +22,9 @@ Also see the [Submitting a bugs/feature request](https://github.com/FreeRTOS/Fre
 **Note:** All the remaining sections are generic and applies to all the versions from V3.0.0 onwards.
 
 ## Upgrading to V3.0.0 and above
```

Comment:
I think we should update this line. This part is valid for V3.0.0 and V3.1.0.
We can reword it as "## Upgrading to V3.0.0 and V3.1.0"

Reply:
Done